extensions/amp-access-fewcents/validator-amp-access-fewcents.protoascii

```diff
@@ -0,0 +1,12 @@
+tags: {  # amp-access-fewcents
+  html_format: AMP
+  tag_name: "SCRIPT"
+  requires_extension: "amp-access"
+  extension_spec: {
+    name: "amp-access-fewcents"
+    version: "0.1"
+    version: "latest"
```

Comment:
I recommend removing this line. We are phasing out `version: "latest"` in new components.

Reply:
@twifkak  Sure, will do that